pkg/kubelet/hyper/hyperclient.go

```diff
@@ -506,12 +338,15 @@ func (c *HyperClient) PullImage(image string, credential string) error {
 		}
 	}
 
+	ctx, cancel := getContextWithTimeout(hyperContextTimeout)
```

Comment:
`hyperContextTimeout` maybe too short for image pulling, use a longer timeout instead for `PullImage`.